server/src/main/java/org/cloudfoundry/identity/uaa/zone/VmidentityIdentityZoneProvisioning.java

```diff
@@ -12,23 +12,25 @@
  *******************************************************************************/
 package org.cloudfoundry.identity.uaa.zone;
 
+import java.io.StringWriter;
+import java.security.PrivateKey;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
+import org.bouncycastle.openssl.PEMWriter;
 import org.cloudfoundry.identity.uaa.constants.OriginKeys;
-import org.cloudfoundry.identity.uaa.provider.VmidentityIdentityProviderProvisioning;
 import org.cloudfoundry.identity.uaa.util.VmidentityUtils;
 
+import com.google.common.primitives.Ints;
```

Comment:
is this something already used by rest of code? otherwise, can we make it without this?